### ISSUE TITLE
Server consider it's crashing on Boombox clicked

### ISSUE THREAD

**Issue by No3371** [Shoxie12/Audiocassettes#2]
Forge 34.1.41

NPE 👀 
It generates crash report for this but continue running.
Could it be you are calling client only methods?
```
[29�Q�@��2020 00:12:58.263] [Server thread/FATAL] [net.minecraft.util.concurrent.ThreadTaskExecutor/]: Error executing task on Server
net.minecraft.crash.ReportedException: Container click
	at net.minecraft.inventory.container.Container.func_184996_a(Container.java:174) ~[?:?]
	at net.minecraft.network.play.ServerPlayNetHandler.func_147351_a(ServerPlayNetHandler.java:1182) ~[?:?]
	at net.minecraft.network.play.client.CClickWindowPacket.func_148833_a(CClickWindowPacket.java:34) ~[?:?]
	at net.minecraft.network.play.client.CClickWindowPacket.func_148833_a(CClickWindowPacket.java:12) ~[?:?]
	at net.minecraft.network.PacketThreadUtil.func_225383_a(SourceFile:21) ~[?:?]
	at net.minecraft.util.concurrent.TickDelayedTask.run(SourceFile:18) ~[?:?]
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213166_h(SourceFile:144) ~[?:?]
	at net.minecraft.util.concurrent.RecursiveEventLoop.func_213166_h(SourceFile:23) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_213166_h(MinecraftServer.java:730) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_213166_h(MinecraftServer.java:156) ~[?:?]
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213168_p(SourceFile:118) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_213205_aW(MinecraftServer.java:713) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_213168_p(MinecraftServer.java:707) ~[?:?]
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213160_bf(SourceFile:103) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_213202_o(MinecraftServer.java:692) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:642) ~[?:?]
	at net.minecraft.server.MinecraftServer.lambda$startServer$0(MinecraftServer.java:229) ~[?:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_261]
Caused by: java.lang.NullPointerException
	at com.shoxie.audiocassettes.slots.BoomBoxSlot.func_75218_e(BoomBoxSlot.java:32) ~[audiocassettes:1.16.3-1.0]
	at net.minecraftforge.items.SlotItemHandler.func_75215_d(SlotItemHandler.java:62) ~[forge:?]
	at net.minecraft.inventory.container.Container.func_75135_a(Container.java:549) ~[?:?]
	at com.shoxie.audiocassettes.container.BoomBoxContainer.func_82846_b(BoomBoxContainer.java:54) ~[audiocassettes:1.16.3-1.0]
	at net.minecraft.inventory.container.Container.func_241440_b_(Container.java:255) ~[?:?]
	at net.minecraft.inventory.container.Container.func_184996_a(Container.java:160) ~[?:?]
	... 17 more
```

**Comment by Shoxie12** [comment]
Yep, you right about client only methods. I need to rethink the way to get the BoomBox tile.
Thank you for the report.

**Comment by Shoxie12** [comment]
Fixed in v1.1
